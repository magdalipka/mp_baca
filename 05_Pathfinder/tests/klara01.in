12 12
1 0 1 0 0 1 0 1 0 0 1 1
0 0 1 0 1 0 0 0 1 0 1 1
0 1 0 0 1 0 1 0 0 0 0 1
0 0 1 0 0 0 1 0 1 1 1 0
1 0 0 0 1 0 1 0 1 0 0 0
0 0 1 1 0 0 1 0 1 0 1 0
0 1 0 0 0 1 1 0 0 0 0 0
1 1 0 1 1 1 0 0 1 0 0 0
0 0 0 1 0 0 0 1 1 0 1 1
0 1 1 1 1 1 1 0 0 0 0 1
1 1 1 1 1 1 1 1 1 1 1 1
1 1 1 1 1 1 1 1 1 1 1 1
10
r 1 11 0 5
r 7 5 11 7
r 7 10 4 5
r 7 6 9 11
r 0 1 8 4
i 1 11 0 5
i 7 5 11 7
i 7 10 4 5
i 7 6 9 11
i 0 1 8 4

