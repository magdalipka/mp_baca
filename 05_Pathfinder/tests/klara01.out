r S W S S E S S W S
r E E S E E N N N
r W W S S S S W S
r N N N E E N N
r X
i S W S S E S S W S
i E E S E E N N N
i W W S S S S W S
i N N N E E N N
i X

